app/src/main/java/org/p2p/wallet/auth/ui/phone/PhoneNumberEnterContract.kt

```diff
@@ -24,7 +24,7 @@ interface PhoneNumberEnterContract {
     interface Presenter : MvpPresenter<View> {
         fun onCountryCodeChanged(newCountryCode: String)
         fun onPhoneChanged(phoneNumber: String)
-        fun onCountryCodeChanged(newCountry: CountryCode)
+        fun onCountryCodeChanged(newCountry: CountryCode?)
```

Comment:
why nullable?

Reply:
special case when we close country picker i send null to open keyboard